src/pages/images/actions/UploadCustomIsoBtn.tsx

```diff
@@ -18,6 +20,8 @@ const UploadCustomIsoBtn: FC<Props> = ({ className, project }) => {
   const { openPortal, closePortal, isOpen, Portal } = usePortal();
   const queryClient = useQueryClient();
   const isSmallScreen = useSmallScreen();
+  const { data: projectObj } = useProject(project);
```

Comment:
```suggestion
  const { data: project } = useProject(projectName);
```
Better rename the incoming prop to projectName and use project for the object. We do this also in other places, and I think it is easier to read.